server/src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -2538,28 +2536,15 @@ public void testCloseOrDeleteIndexDuringSnapshot() throws Exception {
 
         Client client = client();
 
-        boolean allowPartial = randomBoolean();
```

Comment:
Why is that changed?

Reply:
We can't block on the `initialize` part of the snapshot anymore, so I removed the code paths in this test that required blocking on `initialize`. `allowPartial == true` required blocking on init => all the paths that had it set to `true` were removed.